lib/cluster.js

```diff
@@ -107,6 +108,19 @@ Cluster.prototype.getSeedList = function getSeedList(callback) {
     });
 };
 
+Cluster.prototype.getConnectionErrorCount = function getConnectionErrorCount() {
+    return this.connectionErrorCount;
+};
+
+Cluster.prototype.printConnectionErrorMsg = function printConnectionErrorMsg() {
+    var count = this.getConnectionErrorCount();
+    if (count > 0) {
+        console.error('Failed to connect to %d nodes.', count);
+    } else {
+        console.error('Successfully connected to all nodes.');
```

Comment:
Yeah good point. I don't know tbh. I think it's quite useful information that all the nodes have been reached. I print it to stderr so that you can still pipe unix commands.